web/src/features/modals/AnnouncementModal.tsx

```diff
@@ -0,0 +1,71 @@
+import { SourceColorAnnouncementImage } from 'icons/announcementSourceColorImage';
+import { useAtom } from 'jotai';
+import { atomWithStorage } from 'jotai/utils';
+import { useTranslation } from 'react-i18next';
+import { HiXMark } from 'react-icons/hi2';
+import { useSearchParams } from 'react-router-dom';
+import { hasOnboardingBeenSeenAtom } from 'utils/state/atoms';
+
+const hasAnnouncementBeenSeenAtom = atomWithStorage(
+  'announcementSeen',
+  localStorage.getItem('announcementSeen') ?? false
+);
+
+const handleTierGaterClick = (event: React.MouseEvent<HTMLDivElement>) => {
+  event.stopPropagation();
+  event.preventDefault();
+};
```

Comment:
This was the only way that I could get the modal to stay when clicking it, since its parent element has a onClick that makes it dissapper. Suggestions are welcome to do it another way